torchtune/datasets/_slimorca.py

```diff
@@ -17,17 +15,17 @@ def slimorca_dataset(
     *,
     tokenizer: Tokenizer,
     source: str = "Open-Orca/SlimOrca-Dedup",
-    chat_format: Optional[ChatFormat] = Llama2ChatFormat,
+    chat_format: Optional[str] = None,
```

Comment:
Just curious: why do we default to None here now? For better Llama3 support out of the box?

Reply:
yes, llama3 will be used more than llama2 certainly so that should be the default